port/raspberrypi/rp2xxx/src/hal/spi.zig

```diff
@@ -185,6 +186,20 @@ pub const SPI = enum(u1) {
             @compileError("PacketType must be a datatype with a size between 4 and 16 bits inclusive");
     }
 
+    pub fn tx(spi: SPI) dma.DmaWriteTarget {
+        return .{
+            .dreq = if (@intFromEnum(spi) == 0) .spi0_tx else .spi1_tx,
+            .addr = @intFromPtr(&spi.get_regs().SSPDR),
```

Comment:
Don't we not need the `intFromPtr` anymore?

Reply:
addr is u32, so it is needed